docs/src/pages/guides/migration-v4/migration-v4.md

```diff
@@ -1004,6 +1004,25 @@ You can use the [`collapse-rename-collapsedheight` codemod](https://github.com/m
 
   You can use the [`use-transitionprops` codemod](https://github.com/mui-org/material-ui/tree/HEAD/packages/material-ui-codemod#use-transitionprops) for automatic migration.
 
+- Change the default value of `anchorOrigin.vertical` to follow the Material Design guidelines. The menu now displays below the anchor instead of on top of it.
```

Comment:
```suggestion
- Change the default value of `anchorOrigin.vertical` to follow the Material Design guidelines. The menu is not displayed below the anchor instead of on top of it.
```

Reply:
I think this one is already correct. With the new behavior, the menu displays below the anchor.